sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableWriteIT.java

```diff
@@ -83,6 +84,8 @@ public void setup() throws Exception {
         .setClusterId(options.getClusterId())
         .setZoneId(options.getZoneId())
         .setUserAgent("apache-beam-test");
+    bigtableOptionsBuilder = BigtableIO.addBulkOptions(bigtableOptionsBuilder);
+    bigtableOptionsBuilder = BigtableIO.addRetryOptions(bigtableOptionsBuilder);
```

Comment:
these changes should not be necessary -- that implies that our users will have to include them in their code.